source/administration/feature-documentation-template.rst

```diff
@@ -0,0 +1,37 @@
+Feature Documentation Template
+===============================
+
+
+This template defines how a feature should be documented improving presentation
+and retrieval of information of product features. The document is user focused.
+
+Enabling The Feature
+--------------------
+
+Write how the user can enable the feature. The user should utilize the information
+present about the feature and run it.
+
+Defining Use Case
```

Comment:
Wondering why Use Cases would come after enabling the feature?  Seems like use case would be first, describing to the user why they would want to use the feature.